src/main/java/com/nike/cerberus/command/core/UpdateStackCommand.java

```diff
@@ -40,10 +40,11 @@
 public class UpdateStackCommand implements Command {
 
     public static final String COMMAND_NAME = "update-stack";
+    public static final String STACK_NAME_LONG_ARG = "--stack-name";
```

Comment:
Should all these be private?

Reply:
No, these constants are used externally when mapping yaml to args

Reply:
Okay, that explains it.